Makefile

```diff
@@ -51,7 +51,7 @@ POSTGRESQL_CONTROLLER_INTEGRATION_HOST_CONTAINER=postgresql-controler-int-test
 .PHONY: test/integration/postgresql/run
 test/integration/postgresql/run:
 	@echo Running integration test PostgreSQL instance on localhost:5432:
-	-docker run --rm -p 5432:5432 -e POSTGRES_USER=iam_creator --name ${POSTGRESQL_CONTROLLER_INTEGRATION_HOST_CONTAINER} -d timms/postgres-logging:11.5 && sleep 5
+	-docker run --rm -p 5432:5432 -e POSTGRES_USER=admin --name ${POSTGRESQL_CONTROLLER_INTEGRATION_HOST_CONTAINER} -d timms/postgres-logging:11.5 && sleep 5 && echo "CREATE USER iam_creator WITH CREATEDB CREATEROLE VALID UNTIL 'infinity';" | psql -hlocalhost -Uadmin -d postgres
```

Comment:
You still connect with admin right?

Reply:
No. We use `iam_creator` exclusively.

Reply:
Sorry, I refer to that last psql command. You use `admin` to connect. Is that correct?

Reply:
Ah, yes. True. It mimics that some "superuser"  have provided an `iam_creator` role.

The CI step currently fails as `psql` is not available. I'll see if I can use the one in the docker image.

Reply:
https://github.com/lunarway/postgresql-controller/pull/28/checks?check_run_id=340183001#step:7:85